packages/block-library/src/button/editor.scss

```diff
@@ -19,7 +19,7 @@
 	}
 
 	// Add outline to button on focus to indicate focus-state
-	&:focus {
+	&:focus:not(.wp-block-buttons &) {
```

Comment:
What's the purpose of this change?

Reply:
If you check on master, when you focus on a button it shows both outline and box-shadow and it's really weird. It's not related with these changes but makes it look like the other blocks.

Reply:
Okay, but why is this change only being applied to Button blocks nested inside a Buttons block? What makes Button blocks outside of a Buttons block different from those nested in one? And perhaps a comment here would help explain this change, since it's not clear from just reading the CSS.

Reply:
Because the Button (singular) block has an input field, it's annoyingly hard to set focus on the Button directly, but it is possible. Here's how that looks in the main branch:

![master](https://user-images.githubusercontent.com/1204802/100719982-a7ae8500-33bd-11eb-916d-6f05ee64859a.gif)

Here's how that looks in this branch:

![this branch](https://user-images.githubusercontent.com/1204802/100719996-abdaa280-33bd-11eb-80f7-2b3ac60a77cc.gif)

The change seems good. I'd agree it'd be nice to be able to avoid the :not selector. Could a rule exist in the Buttons block editor style instead?